profiles/common/themes/europa/Gruntfile.js

```diff
@@ -39,7 +44,7 @@ module.exports = function (grunt) {
     copy: {
       main: {
         files: [
-          // includes files within path and its sub-directories
+          // Includes files within path and its sub-directories
```

Comment:
Inline comments must end with a period, exclamation mark or question mark. Strange that this is not detected. There is a test for this in `InlineCommentSniff` which works for PHP code, but this doesn't seem to work in Javascript.

I've created an issue for this: https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-5456